src/registryctl/main.go

```diff
@@ -58,7 +70,7 @@ func (s *RegistryCtl) Start() {
 
 func main() {
 
-	configPath := flag.String("c", "", "Specify the yaml config file path")
+	configPath := flag.String("c", "", "Specify the yaml rConf file path")
```

Comment:
Ideally there should be a config file of registryctl that points to the registry configuration file.

Reply:
updated.